lua/wikis/commons/Infobox/Extension/TeamHistory/Auto.lua

```diff
@@ -23,7 +24,8 @@ local TeamHistoryAuto = {}
 ---@return Widget?
 function TeamHistoryAuto.run(frame)
 	local args = Arguments.getArgs(frame)
-	local player = args.player or String.upperCaseFirst(mw.title.getCurrentTitle().subpageText)
+	local player = Page.applyUnderScoresIfEnforced(args.player
+		or String.upperCaseFirst(mw.title.getCurrentTitle().subpageText))
```

Comment:
Do we really need the uppercasefirst here?

Reply:
i copied it from the old module
i think the usecase is for subages in userspace
like if i would have `User:Hjpalpha/eonBlu` (because his displayName used to be `eonBlu`) it would not work without the `String.upperCaseFirst`

Reply:
We could argue that we shouldn't cater to userspace, but whatever ^